src/hooks/HMSRoomProvider.tsx

```diff
@@ -174,3 +177,8 @@ export const useHMSNotifications = () => {
 
   return notification;
 };
+
+export const useIsHMSStatsOn = () => {
```

Comment:
Possible to make this selector based? getState(selectHMSStats.isStatsOn)?
Maybe we can always create and return an object with an additional call - hmsActions.enableStats(config?: {localPeerStats: boolean}) to actually enable. This should simplify some things and make the turning on stats explicit that stats are on as well as give a way to disable them again. 
In future with remote stats, not everyone might be interested in remote network level or remote stats, where the config can be used to enable/disable things as required.

Reply:
Created this selector for internal use only(to use in VideoTile), the app would know this since `isHMSStatsOn` or `statsStore` prop is passed to HMSRoomProvider from app side.

The config part would require some major changes in the SDK and also actions will take in separate PR?

Reply:
Sure, we'll remove passing in from the app once we completely migrate over to the new repo btw.